var/spack/repos/builtin/packages/py-cdo/package.py

```diff
@@ -12,10 +12,21 @@ class PyCdo(PythonPackage):
 
     pypi = "cdo/cdo-1.3.2.tar.gz"
 
-    version("1.3.2", sha256="9f78879d90d14134f2320565016d0d371b7dfe7ec71110fd313868ec1db34aee")
+    maintainers = ["Try2Code", "skosukhin"]
 
-    depends_on("cdo")
+    version("1.5.6", sha256="fec1a75382f01b3c9c368e8f143d98b12323e06975663f87d9b60c739ae1d335")
+    version(
+        "1.3.2",
+        sha256="9f78879d90d14134f2320565016d0d371b7dfe7ec71110fd313868ec1db34aee",
+        deprecated=True,
+    )
 
     depends_on("py-setuptools", type="build")
-    depends_on("py-scipy", type=("build", "run"))
-    depends_on("py-netcdf4", type=("build", "run"))
+    depends_on("cdo+netcdf", type="run")
+    depends_on("py-netcdf4", type="run")
+    depends_on("py-scipy", type="run", when="@:1.4")
+    depends_on("py-xarray", type="run", when="@1.3.4:")
+    depends_on("py-six", type="run", when="@1.3.3:")
```

Comment:
These should all be `type=('build', 'run')` so that the import tests work properly

Reply:
Ok. I did that. For the future, what command fails if I don't do that? I tried `spack install --test=root py-cdo` and it worked without this change.

Reply:
That should trigger it, maybe another dependency is adding them to `PYTHONPATH`, but not a great behavior to rely on.

Reply:
@adamjstewart ok but why should they be `build` and not `test` then?

Reply:
Hmm, I think you might be right, they should technically be `type=('test', 'run')`. But every other package uses `type=('build', 'run')` at the moment. Personally I would prefer to be consistent. Someday, I would like Spack to be smart enough to realize that post-install tests ARE run-time tests, so env vars should be modified accordingly. Maybe @tldahlgren can comment on this.